### DISCUSSION TITLE
E-mail

### DISCUSSION THREAD

**Discussion by lucacaputo09** [iisgiua/giuaschool#696] [Richieste di aiuto] [answered]
Salve, è possibile modificare il testo della mail quando arrivano le credenziali al destinatario?

**Comment by trinko** [comment, marked as answer]
Devi modificare i template relativi alle email.
Li trovi nella cartella: **templates/email/**
Ma non modificare direttamente quelli, copiali in **PERSONAL/templates/email/** e modifica la copia.
In questo modo non verranno sovrascritti durante gli aggiornamenti.
